src/cli2/load.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+
+import type {Command} from "./command";
+import {loadInstanceConfig, pluginDirectoryContext} from "./common";
+
+function die(std, message) {
+  std.err("fatal: " + message);
+  return 1;
+}
+
+const loadCommand: Command = async (args, std) => {
+  if (args.length !== 0) {
+    die(std, "usage: sourcecred load");
+  }
+  const baseDir = process.cwd();
+  const config = await loadInstanceConfig(baseDir);
+  for (const [name, plugin] of config.bundledPlugins) {
+    const dirContext = pluginDirectoryContext(baseDir, name);
+    plugin.load(dirContext);
```

Comment:
Let's have the task reporter report that load started for the plugin--that way we have consistent measurements for all the plugins, and the plugin can use the provided task reporter to give more detailed information (e.g. timing on sub-portions)